webservice/routes/api.php

```diff
@@ -3,11 +3,13 @@
 Route::group([
     'middleware' => ['cors', 'api'],
 ], function () {
-    Route::post('/login', 'LoginController@login');
+    Route::post('/auth/issue', 'AuthController@issueToken');
```

Comment:
I don't know... 
We don't have another auth method, I prefer just `POST /auth`

But, if we follow REST `POST /auth/token` make more sense